src/features/areas/components/PublicAreaMap.tsx

```diff
@@ -115,18 +119,12 @@ const PublicAreaMap: FC<PublicAreaMapProps> = ({ area }) => {
     const crosshair = crosshairRef.current;
 
     if (map && crosshair) {
-      const mapContainer = map.getContainer();
-      const markerRect = crosshair.getBoundingClientRect();
-      const mapRect = mapContainer.getBoundingClientRect();
-      const x = markerRect.x - mapRect.x;
-      const y = markerRect.y - mapRect.y;
-      const markerX = x + 0.5 * markerRect.width;
-      const markerY = y + 0.5 * markerRect.height;
+      const markers = getMarkerValues(map, crosshair);
```

Comment:
The plural name here (`markers`) indicates to me that the return value from `getMarkerValues()` is somehow an array or some other plural thing, but it's not. I think the name of this variable should be `markerPos` or something?